JFrogVSExtension/Utils/Util.cs

```diff
@@ -219,19 +254,25 @@ public static HashSet<Components> GetComponents(Dependency[] dependencies, HashS
                 {
                     component_id = PREFIX + dependency.id
                 };
+
                 if (!componentsCache.Contains(comp))
                 {
                     ids.Add(comp);
-                    if (dependency.dependencies != null && dependency.dependencies.Length > 0)
-                    {
-                        HashSet<Components> internalIdS = GetComponents(dependency.dependencies, componentsCache);
-                        ids.UnionWith(internalIdS);
-                    }
                 }
 
+                // Discover comp's dependencies even if already exists in cache.
+                // This is due to a possible situation of multiple subprojects under the same solution,
+                // where the same package appears in several projects but its dependencies varies between the packages.config of the subprojects.
```

Comment:
Let's also rephrase here, as I suggested in a different comment.

Reply:
Changed.